webpack/ForemanTasks/Components/TaskActions/TaskActionHelpers.js

```diff
@@ -29,16 +30,40 @@ export const convertDashboardQuery = query => {
 };
 
 export const resumeToastInfo = {
-  resumed: { type: 'success', text: __('was resumed') },
-  failed: { type: 'error', text: __('could not be resumed') },
-  skipped: { type: 'warning', text: __('task has to be resumable') },
+  resumed: { type: TOAST_TYPES.SUCCESS, text: __('was resumed') },
+  failed: { type: TOAST_TYPES.ERROR, text: __('could not be resumed') },
+  skipped: { type: TOAST_TYPES.WARNING, text: __('task has to be resumable') },
 };
 
 export const cancelToastInfo = {
-  cancelled: { type: 'success', text: __('was cancelled') },
-  skipped: { type: 'warning', text: __('task has to be cancellable') },
+  cancelled: { type: TOAST_TYPES.SUCCESS, text: __('was cancelled') },
```

Comment:
we are using here those `{type, text}` pairs many times,
can you ad another helper, e.g. `warningToastInfo('some-text')` which only receive text and return this object?
same for `success, error`

Reply:
Great idea! changed